D/Doom/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+
+using BinaryBuilder, Pkg
+
+name = "Doom"
+version = v"0.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/AlexOberhofer/sdl2-doom.git",
+              "da7732ee6318371db2ee04ec4702c6064245846b"),
+    DirectorySource("bundled"),
+    FileSource("https://distro.ibiblio.org/slitaz/sources/packages/d/doom1.wad",
+               "1d7d43be501e67d927e415e0b8f3e29c3bf33075e859721816f652a526cac771"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/sdl2-doom
+atomic_patch -p1 ../patches/11.diff
+cd src
+if [[ "${target}" == *-mingw* ]]; then
+    make -f makefile.mingw -j${nproc} CC="${CC}"
+else
+    make -j${nproc}
+fi
+install -Dvm 755 "sdl2-doom${exeext}" "${bindir}/doom${exeext}"
+cd ..
+install -Dvm 644 "${WORKSPACE}/srcdir/doom1.wad" -t "${prefix}/share/doom"
```

Comment:
Add it as a FileProduct so that (1) its presence is ensured and (2) it's easier to reference it instead of guessing its path?

Reply:
So do I keep that install line for doom1.wad and just add a FileProduct for it, which will then let me use the variable name to pull it up rather than the full path?